XenModel/Alerts/PerfmonOptionsDefinition.cs

```diff
@@ -117,11 +197,12 @@ public static PerfmonOptionsDefinition GetPerfmonOptionsDefinitions(IXenObject x
 
             string mailDestination = GetMailDestination(connection);
             string mailHub = GetSmtpMailHub(connection);
+            string mailLanguageCode = GetMailLanguageCode(connection);
 
-            if (mailDestination == null || mailDestination == null)
+            if (mailDestination == null || mailHub == null || mailLanguageCode == null)

```

Comment:
We can only force not null mailLanguageCode on the pools where this featue is implemented. Since we are going to do the version/feature implementation check later, we should probably remove the `mailLanguageCode == null` check for now.